src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { applyMiddleware, legacy_createStore as createStore, combineReducers } from 'redux';
+import thunk from 'redux-thunk';
```

Comment:
- [ ] Please know that you need to use `@reduxjs/toolkit` properly here, by importing `configureStore` from it then using it to configure your store, kindly change this.